index.js

```diff
@@ -1,9 +1,11 @@
 const generateId = () => (Math.random() + 1).toString(36).substring(7);
 
```

Comment:
- [ ] You should use JavaScript classes for your bookCollection instead of objects and arrays (You can use objects and arrays, but do it inside classes)